src/Bicep.Cli.IntegrationTests/BuildParamsCommandTests.cs

```diff
@@ -348,6 +348,8 @@ public async Task Build_params_to_stdout_with_non_bicep_references_should_succee
             result.Should().Succeed().And.NotHaveStderr();
 
             var parametersStdout = result.Stdout.FromJson<BuildParamsStdout>();
+            // Force consistency for escaped newlines.
+            parametersStdout = parametersStdout with { templateJson = parametersStdout?.templateJson?.Replace("\r\n", "\n") };
```

Comment:
[nit]: There's a built-in method to do this: https://learn.microsoft.com/en-us/dotnet/api/system.string.replacelineendings?view=net-8.0#system-string-replacelineendings